modules/openapi-generator/src/main/resources/kotlin-client/interface_req_var.mustache

```diff
@@ -5,9 +5,6 @@
     {{#moshi}}
     @Json(name = "{{{vendorExtensions.x-base-name-literal}}}")
     {{/moshi}}
-    {{#gson}}
```

Comment:
@kuFEAR same here

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/5750#issuecomment-626663248 1.

Reply:
Thanks

Reply:
I have an idea add @get:SerializedName("field")

Reply:
If that works, it would be nice, because I use Gson on my projects

Reply:
\+

Reply:
Thanks